lib/tests/streamlit/streamlit_test.py

```diff
@@ -428,7 +428,7 @@ def test_st_pyplot(self):
 
         # Manually calculated by letting the test fail and copying and
         # pasting the result.
-        checksum = "DTuIkOADCFAAEAmPL/AFE92BIZHj8WAAAAAElFTkSuQmCC"
+        checksum = "n1ZpaandJQILIkAAAJ7EQ3QAgCkECADAlP8PvwXGkVVfFJkAAAAASUVORK5CYII="
```

Comment:
Why is this changed? You didn't touch pyplot in you code, so the test shouldn't fail unless something is actually broken.